### PR TITLE
(365) YAML guidance and hint text

### PR DESCRIPTION
**Pull request by jkempster34**
### Fix spelling of received/receive
IDE highlighted this so I thought I might as well fix it while I was there

### Add hint and guidance to actions
We want to offer users some additional hints and guidance. 

Update the models to include the new columns, and update the conversions workflow to match.

### Render hints and guidance
For now, render the hint as part of the checkbox, and conditionally render the guidance.

The guidance (rendered as a details component) currently sits within the checkbox fieldset (because we have modelled actions checkboxes within a fieldset). I can imagine this will change as move move toward each action being a distinct thing.

## Checklist

- [x] Attach this pull request to the appropriate card in Trello.
- [x] Update the `CHANGELOG.md` if needed.

### CODE REVIEW

**Comment by jkempster34** [comment]
[Add Hint text and Guidance text to Actions](https://trello.com/c/rDmkVR1w/365-add-hint-text-and-guidance-text-to-actions)

**Comment by mec** [comment]
Booted this up, it's not on the card but I think we need to support white space and probably bullets as well - these are not currently working?

Screenshot 2022-08-16 at 10-57-32 Complete conversions transfers and changes

**Comment by jkempster34** [comment]
> Booted this up, it's not on the card but I think we need to support white space and probably bullets as well - these are not currently working?
> 
> Screenshot 2022-08-16 at 10-57-32 Complete conversions transfers and changes

Just added some guidance to the README. How does that look?

File: app/models/action.rb [hunk 1]
```diff
@@ -2,4 +2,7 @@ class Action < ApplicationRecord
   belongs_to :task
 
   default_scope { order(order: "asc") }
+
+  validates :guidance_summary, presence: true, if: -> { guidance_text.present? }
```
Comment (mec) [inline app/models/action.rb:6]: I am not sure we need to validate this - I guess it is okay, but feels a bit too much - less code is always better in my head. I generally see validation as a user input thing, which this is not.

File: app/workflows/conversion.yml [hunk 2]
```diff
@@ -66,6 +66,9 @@ sections:
       - title: Land questionnaire & land registry title plans
         actions:
           - title: Land questionnaire received
+            hint: Hello, I'm a hint
```
Comment (mec) [inline app/workflows/conversion.yml:69]: Total nitpick, but I would like to see something closer to real copy here and evidence of all the formatting we support.

File: spec/services/task_list_creator_spec.rb [hunk 3]
```diff
@@ -35,11 +35,20 @@
...
 
     it "creates actions from the workflow" do
-      section = Section.find_by(title: "Clear legal documents")
-      task = section.tasks.find_by(title: "Clear land questionnaire")
+      section = Section.find_by(title: "Starting the project")
+      task = section.tasks.find_by(title: "Understand history and complete handover from Pre-AB")
 
       expect(Action.count).to eql 6
-      expect(Action.where(title: "Action one", order: 0, task: task)).to exist
+      expect(
```
Comment (mec) [inline spec/services/task_list_creator_spec.rb:42]: Do you think this warrants a feature spec around helping users? Something like `Users can see hint text on Actions` and `Users can get additional guidnace on actions` - which could include tests for basic formating?

File: README.md [hunk 4]
```diff
@@ -26,8 +25,25 @@ See .env.example
 Use the [release process template in Trello](https://trello.com/c/8enGdMyy) to
 start a new release.
 
+## Including Markdown in the workflow
```
Comment (mec) [inline README.md:28]: :heart: this! I would like to split this out into it's own file and link to it, but not now!

**Pull request merged**